samples/web/content/apprtc/apprtc.py

```diff
@@ -288,35 +288,42 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
   media_constraints = make_media_stream_constraints(audio, video,
                                                     firefox_fake_device)
   wss_url, wss_post_url = get_wss_parameters(request)
+  jsonParams = json.dumps({
```

Comment:
This will break mobile clients which depend on the output of get_room_parameters. Why do you have to change this?